### PR TITLE
VBM-107: mocked user removed

### PR DESCRIPTION
**Pull request by VladimirBludov**

### CODE REVIEW

File: packages/mobile/src/helpers/user-parser/auth-user-parser.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { UserDto, UserResponseDto } from '@vse-bude/shared';
+
+const authUserParser = (user: UserResponseDto): UserDto => {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/helpers/user-parser/auth-user-parser.ts:3]: why do we need this?

File: packages/mobile/src/screens/personal-info/components/personal-info-form/personal-info-form.tsx [hunk 2]
```diff
@@ -33,21 +29,22 @@ type Props = {
...
-      linkedin: TEST_USER.linkedin,
-      facebook: TEST_USER.facebook,
-      password: TEST_USER.password,
+      firstName: user?.firstName || '',
+      lastName: user?.lastName || '',
+      email: user?.email || '',
+      phone: user?.phone || '',
+      country: 'none',
+      region: 'none',
+      city: 'none',
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/screens/personal-info/components/personal-info-form/personal-info-form.tsx:41]: Can we set these to be empty strings by default too?

Comment (VladimirBludov) [reply packages/mobile/src/screens/personal-info/components/personal-info-form/personal-info-form.tsx:41]: yes, corrected

File: packages/mobile/src/helpers/user-parser/auth-user-parser.ts (hunk 1 above)
Comment (VladimirBludov) [reply packages/mobile/src/helpers/user-parser/auth-user-parser.ts]: removed

**Pull request merged**